github/ci/prow-deploy/kustom/base/configs/current/labels/labels.yaml

```diff
@@ -261,6 +261,23 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - name: sig/scale
+      color: aefcf3
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - name: sig/buildsystem
+      color: 73efeb
+      description: Denotes an issue or PR that relates to changes in the build system, touching either components or configuration of it.
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - name: sig/api
```

Comment:
We indeed have a  SIG API, so this makes sense.
Just note that there are two other labels that somehow "compete" with this one.
- `kind/api-change`
- `topic/api`

We need to do some cleanup or learn how to use these better.
The `topic/api` should probably be deprecated.

Reply:
Yes, I agree, and I think this will be part of the follow up discussions happening soon.